TEP003_commit_message_guidelines.rst

```diff
@@ -0,0 +1,119 @@
+TEP003: Git Policy
+========================
+
+Status
+======
+
+ **Discussion**
+
+Responsible
+===========
+
+@yeganer, @orbitfold
+
+Description
+===========
+
+Commit Message Guidelines
+-------------------------
+
+When committing changes do not use -m option. This is only suitable for really short commit messages. Instead omit the message flag and you will be taken to your default text editor where you can enter the message. If you have already pushed the changes to a remote repository or if you have committed a change locally with a message that does not conform to the guidelines given below please use the following instructions to change commit messages:
+
+https://help.github.com/articles/changing-a-commit-message/
+
+The following are the requirements for the commit messages:
+
+* The commit message consists of the following sections:
+
+  * One-line description
+  * Empty line
+  * Body of the commit message
+
+* One-line description consists of the following:
```

Comment:
Ich think we should emphasize to write in imperative form. In my opinion the title should complete the sentence
*If applied this commit will ...* Fix bug #12345